common/live/prd/infra/dev/zone/terraform.tfvars

```diff
@@ -22,4 +22,9 @@ project_name    = "dev"
 # of the role:
 # https://www.terraform.io/docs/providers/google/r/google_project_iam.html#argument-reference
 
-project_owner   = "group:gpii-bot@raisingthefloor.org"
+# root projects: dev,stg and prd don't need a project_owner. They will use the
+# IAMs inherited from the org.
+#
+# The variable is set to avoid a failure in the execution of the module, but it won't be set.
+
+project_owner   = "group:cloud-admin@raisingthefloor.org"
```

Comment:
Can we perhaps make this as a default, and not set it here - and set to some non-sense value? Smth. like `does-not-exist@...`... I think it would be more obvious anshould work untill we actually try to assign policy with that user to smth. (additional safety check that it's really not used)